packages/gestalt/src/ExpandableModuleBase.js

```diff
@@ -0,0 +1,112 @@
+// @flow strict
+import React, { type Node } from 'react';
+import Box from './Box.js';
+import IconButton from './IconButton.js';
+import Icon from './Icon.js';
+import TapArea from './TapArea.js';
+import Text from './Text.js';
+
+type Props = {|
+  title: string,
+  icon?: string,
+  arrowIconAccessibilityLabel: string,
+  titleIconAccessibilityLabel?: string,
+  summary?: Array<string>,
+  isCollapsed: boolean,
+  onModuleClicked?: boolean => void,
+  type: 'error' | 'info',
+  children?: Node,
+|};
+
+const EXPANDABLE_TYPE_ATTRIBUTES = {
+  info: {
+    color: 'darkGray',
+  },
+  error: {
+    icon: 'workflow-status-problem',
+    color: 'red',
+    accessibilityLabel: 'Error icon',
+  },
+};
+
+export default function ExpandableModuleBase({
+  title,
+  icon,
+  arrowIconAccessibilityLabel,
+  titleIconAccessibilityLabel,
+  summary,
+  isCollapsed,
+  onModuleClicked,
+  type,
+  children,
+}: Props): Node {
+  const handleModuleClick = () => {
+    if (onModuleClicked) {
+      onModuleClicked(!isCollapsed);
+    }
+  };
+  return (
+    <>
+      <TapArea onTap={handleModuleClick}>
```

Comment:
Please use `accessibilityExpanded` on `<TapArea>` to conform with the WAI-ARIA Design Pattern of "Accordion" from https://www.w3.org/TR/wai-aria-practices-1.1/#accordion

Reply:
More info on https://gestalt.netlify.app/TapArea#accessibility

Something else that can be considered is to add `accessibilityControls` as it's also part of WAI-ARIA Design Pattern of "Accordion" but that also requires giving an `id` to the content area and that may overcomplicate things here, so I'd leave that just for thought.